packages/desktop-client/src/components/reports/reports/Calendar.tsx

```
@@ -0,0 +1,946 @@
+    onUpdate: onUpdateFilter,
+    onConditionsOpChange,
+  } = useFilters(widget?.meta?.conditions, widget?.meta?.conditionsOp);
+
+  useEffect(() => {
+    const day = parameters.get('day');
+    const month = parameters.get('month');
+
+    if (day && onApplyFilter) {
+      onApplyFilter({
+        conditions: [
+          ...widget?.meta?.conditions,
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)